fabfile.py

```diff
@@ -71,6 +71,10 @@ def pdf(language='en'):
             local('pdflatex -interaction=nonstopmode Mixxx-Manual.tex')
             local('pdflatex -interaction=nonstopmode Mixxx-Manual.tex')
 
+@task
+def versionedhtml(language='en'):
+    local('%s versionedhtml' % make_command(language))
+
```

Comment:
looks like the versionedpdf command is now gone?

Reply:
Yeah it was moved to the PDF change branch.

Reply:
Gotcha -- can you add it back here? In order to check this in we need the PDF and HTML build succeeding for all versions (since the Jenkins job that runs this will delete everything at https://mixxx.org/manual/ on every build)

Reply:
(Sorry it was ambiguous when I asked to split out the engine changes -- I was just talking about the xetex change)

Reply:
That won't really work, at least for all languages except English. I'm not able to build these without xetex due to unicode chars that cause issues.

Reply:
> That won't really work, at least for all languages except English. I'm not able to build these without xetex due to unicode chars that cause issues.

Right, but your machine isn't doing the building :). 

The PDF build succeeds for a variety of languages without any engine changes:

https://builds.renegadetech.mixxx.org/job/v2.2/job/manual-2.2-publish/ws/languages/de-DE/Mixxx-Manual.pdf/*view*/
https://builds.renegadetech.mixxx.org/job/v2.2/job/manual-2.2-publish/ws/languages/it/Mixxx-Manual.pdf/*view*/

etc.

Reply:
Ok, I think I removed everything related to XeTeX.